src/main/java/net/mcreator/element/types/Biome.java

```diff
@@ -65,6 +65,22 @@
 	public double baseHeight;
 	public double heightVariation;
 
+	//m stands for multi-noise
+	public boolean usePureMultiNoise;
+	public double mTemperature1;

```

Comment:
Replace your template for something like `minTemperature`, `maxTemperature`. It will be more understandable.

Reply:
This was fixed in "First Fix" (well, at least I hope so) (actually, thanks for this tip, I have no idea why I didnt named them like this in a first place)